testmain_test.go

```diff
@@ -100,3 +161,62 @@ func envFlagString(envName, name, value, usage string) *string {
 func DefaultOptions() *cbcolumnar.ClusterOptions {
 	return cbcolumnar.NewClusterOptions().SetSecurityOptions(cbcolumnar.NewSecurityOptions().SetDisableServerCertificateVerification(true))
 }
+
+var globalTestLogger *testLogger
+
+type testLogger struct {
+	Parent           cbcolumnar.Logger
+	LogCount         []uint64
+	suppressWarnings uint32
+}
+
+func (logger *testLogger) Log(level cbcolumnar.LogLevel, offset int, format string, v ...interface{}) error {
+	if level >= 0 && level < cbcolumnar.LogMaxVerbosity {
+		if atomic.LoadUint32(&logger.suppressWarnings) == 1 && level == cbcolumnar.LogWarn {
+			level = cbcolumnar.LogInfo
```

Comment:
If 'atomic.LoadUint32(&logger.suppressWarnings) == 1` doesn't that mean that we want to suppress warning logs so we should change the log level to Error here?

Reply:
No, we're suppressing the logging so that it doesn't cause the tests to fail.